docs/graphql/manual/migrations/reference/metadata-format.rst

```diff
@@ -25,12 +25,239 @@ of multiple files.
+                  author_name
+              }
+          }
+    
+In above example there is only one collection ``allowed_queries`` which contains two queries ``test`` and ``test2`` .
+
+
+allow_list.yaml
+^^^^^^^^^^^^^^^
+
+Contains the metadata related to :ref:`allow lists<allow_list>`
+
+..  code-block::  yaml
+
+  - collection: allowed-queries
+
+The allowed queries are under the collection name ``allowed-queries``
+``allow_list.yaml`` contains collection name of allowed queries.
+
+
+
+actions.yaml
+^^^^^^^^^^^^
+
+Contains the metadata related to :ref:`actions<actions>`
+
+..  code-block::  yaml
+
+      actions:
+    - name: greet
+      definition:
+        kind: ""
+        handler: <base_url>/greet
+        forward_client_headers: true
+        headers:
+        - value: application/json
+          name: Content-Type
+    custom_types:
+      enums: []
+      input_objects:
+      - name: SampleInput
+      objects:
+      - name: SampleOutput
+      scalars: []
+
+
+In above example the url is specified directly to handler field
+
+..  code-block::  yaml
+
+      actions:
+    - name: greet
+      definition:
+        kind: ""
+        handler: '{{ACTION_BASE_URL}}/greet'
+        forward_client_headers: true
+        headers:
+        - value: application/json
+          name: Content-Type
+    custom_types:
+      enums: []
+      input_objects:
+      - name: SampleInput
+      objects:
+      - name: SampleOutput
+      scalars: []
+
+
+The above example is same to the previous except the url is specified directly not via environment variable.
+
+actions.graphql
+^^^^^^^^^^^^^^^
+
+Contains all the action definition and custom type definitions where the metadata information about actions will be stored in actions.yaml
+
+..  code-block:: graphql
+
+      type Query {
+        greet (
+          arg1: SampleInput!
+        ): SampleOutput
+      }
+      input SampleInput {
+        username : String!
+      }
+      type SampleOutput {
+        greetings : String!
+      }
+
+
+The above example actiions.graphql contains the definition of greet action and custom type  SampleOutput and SampleInput definitions.
+
+cron_triggers.yaml
+^^^^^^^^^^^^^^^^^^
+
+Contains metadata related to cron triggers
+
+..  code-block  yaml
+
+      - name: test
+          webhook: <webhook_url>
+          schedule: 0 12 * * 1-5
+          include_in_metadata: true
+          payload: {}
+          retry_conf:
+            num_retries: 1
+            timeout_seconds: 60
+            tolerance_seconds: 21600
+            retry_interval_seconds: 10
+
+In above example ``cron_triggers.yaml`` contains the information about the test cron trigger. Here webhook_url can be http endpoint that should be triggered or environment variable
+which has http endpoints that should be triggered.
+
+..  note::
+  
+  The metadata about cron triggers will not be stored if ``Include this trigger in Hasura Metadata`` is disabled in advanced option of event in console.
```

Comment:
The metadata about cron triggers will not be stored if ``Include this trigger in Hasura Metadata`` is disabled in the advanced option of ``events`` on the console.